lib/src/collections.dart

```diff
@@ -0,0 +1,23 @@
+import 'booleans.dart';
+import 'objects.dart';
+
+/// True if the provided [Iterable] or [Map] is not null and is
+/// [Iterable.isNotEmpty]
```

Comment:
Add grave around null (`null`) and remove double "is"
```suggestion
/// Returns `true` if the provided [Iterable] or [Map] is not `null` and [Iterable.isNotEmpty]
```